README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
- [ ] I think it's better if we use the README file from [Microverse Template](https://github.com/microverseinc/readme-template). Although the one from CRA is not incorrect, it does seem generic and doesn't document the project in a very professional way. I do recommend as well including the scripts from this document in the other.

Reply:
working on it :+1: